lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -117,9 +117,62 @@ module.exports = [
         'font-size': {
           score: 1,
           details: {
-            items: {
-              length: 6,
-            },
+            items: [
+              {
+                source: 'http://localhost:10200/seo/seo-tester.html?extra_header=link%3D%253Chttp%253A%252F%252Flocalhost%253A10200%252Fseo%252F%253E%253B%2Brel%253D%2522canonical%2522:24:12',
```

Comment:
yikes this is hard to tell what the important assertion is, WDYT about regex with `/seo-tester.*24:12$/` :D

Reply:
i was gonna do `/24:12$/` at first but saw that it'd be hard to tie that to a particular URL. didn't consider `/seo-tester.*24:12$/` - i like that